example/lib/data.dart

```diff
@@ -0,0 +1,4 @@
+List<String> get data => [
+      "https://storage.cloud.google.com/ticketma-dev.appspot.com/passes/10911/6923491/pass_0.pkpass",
```

Comment:
Access to this asset is forbidden.
Think about moving it into some other place (f.e drive google with full access to view).

Reply:
Sure, I didn't use it anyway I implemented it directly to the app